Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.js

```diff
@@ -0,0 +1,150 @@
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+ * BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+ * THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+WI.OverrideUserPreferencesPopover = class OverrideUserPreferencesPopover extends WI.Popover
+{
+    constructor(delegate)
+    {
+        super(delegate);
+
+        this._targetElement = null;
+        this._defaultValueOptionElementsMap = new Map;
+        this.windowResizeHandler = this._presentOverTargetElement.bind(this);
+    }
+
+    // Public
+
+    show(targetElement)
+    {
+        this._targetElement = targetElement;
+        this.content = this._createContentElement();
+
+        WI.cssManager.addEventListener(WI.CSSManager.Event.DefaultUserPreferencesDidChange, this._defaultUserPreferencesDidChange, this);
+
+        this._presentOverTargetElement();
+    }
+
+    dismiss()
+    {
+        this._defaultValueOptionElementsMap.clear();
+
+        super.dismiss();
+    }
+
+    // Private
+
+    _presentOverTargetElement()
+    {
+        if (!this._targetElement)
+            return;
+
+        let targetFrame = WI.Rect.rectFromClientRect(this._targetElement.getBoundingClientRect());
+        const preferredEdges = [WI.RectEdge.MAX_Y, WI.RectEdge.MAX_X];
+
+        this.present(targetFrame.pad(2), preferredEdges);
+    }
+
+    _createSelectElement(contentElement, label, preferenceName, values, defaultValue)
+    {
+        if (!preferenceName || !values.length)
+            return;
+
+        let labelElement = contentElement.appendChild(document.createElement("label"));
+        labelElement.textContent = label;
+        labelElement.setAttribute("for", preferenceName);
+
+        let selectElement = contentElement.appendChild(document.createElement("select"));
+        selectElement.setAttribute("name", preferenceName);
+        selectElement.setAttribute("id", preferenceName);
+
+        let systemValue = WI.cssManager.defaultUserPreferences.get(preferenceName);
+
+        for (let value of [defaultValue, ...values]) {
+            let optionElement = selectElement.appendChild(document.createElement("option"));
+            optionElement.value = value;
+            optionElement.textContent = this._userPreferenceValueLabel(value, systemValue);
+
+            let selectedValue = WI.cssManager.overridenUserPreferences.get(preferenceName) || defaultValue;
+            if (value === selectedValue)
+                optionElement.setAttribute("selected", true);
+
+            if (value === defaultValue)
```

Comment:
I think we should also add a `<hr>` after the "System (...)" `<option>` so that it's even clearer that it's not one of the actual values (this is what we do for the Appearance dropdown in the Settings Tab).

Reply:
Done